config/locales/en.yml

```diff
@@ -55,7 +77,22 @@ en:
           country_of_origin: "Country of origin"
           racial_ethnic_identity: "Racial/ethnic identity"
           sexual_orientation: "Sexual orientation"
+          sexual_orientation_options:
```

Comment:
these ones are in a completed story and simone said they will always be displayed in english (sexual_orientation_options and gender_options)

Reply:
I second this. Though, I think it is worthwhile getting these phrases translated since they might not be widely known phrases and the assisters may not know them off-hand

Reply:
thanks for that! I'll remove them